api/audit-trail-gw/src/routes/channel/index.ts

```diff
@@ -19,14 +19,14 @@ export class ChannelRoutes {
 
 	createChannel = async (req: AuthenticatedRequest, res: Response, next: NextFunction): Promise<Response<any>> => {
 		try {
-			const { topics, seed, hasPresharedKey, presharedKey, subscriptionPassword } = req.body as CreateChannelBody;
+			const { name, topics, seed, hasPresharedKey, presharedKey } = req.body as CreateChannelBody;
```

Comment:
Channel name should be unique. Please add a check as it is done when creating an identity. 

In addition please seed the database accordingly as done for the username here:
https://github.com/iotaledger/integration-services/blob/develop/api/ssi-bridge/src/setup/database-seeder.ts#L19
But use the seeder of the audit-trail-gw